pkg/sentry/arch/arch_aarch64.go

```diff
@@ -32,26 +32,26 @@ import (
 const (
 	// SyscallWidth is the width of insturctions.
 	SyscallWidth = 4
+
+	// fpsimd_magic is the magic number which is used in fpsimd_context.
```

Comment:
Please update the names in the comments to match the actual name.

Reply:
Done.